src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_InversionFaultSystemRuptSet.java

```diff
@@ -68,8 +77,10 @@ public NZSHM22_SubductionInversionTargetMFDs call() throws Exception {
 	}
 
 	public static NZSHM22_InversionFaultSystemRuptSet fromCrustal(FaultSystemRupSet rupSet, U3LogicTreeBranch branch){
-		branch.clearValue(ScalingRelationships.class);
-		branch.setValue(ScalingRelationships.TMG_CRU_2017);
+		if(ScalingRelationships.SHAW_2009_MOD == branch.getValue(ScalingRelationships.class)) {
```

Comment:
@voj do we need/want this? e.g. For Crustal SHAW_2009 is a valid option. Let's just do what the user decides is best, no matter how weird ;)

Reply:
Oh - I see this is in 'fromSubduction'. Even so,  I don't want to silently override user inputs.

Reply:
It's not perfect, I think a proper solution will require us to decide on how we do configurations/logic tree branches. This is already better than before when we would always override any possible user's choice. I think we should wait until we sort out configurations to fix this.